tests/helpers/WebhookProxy.ts

```diff
@@ -40,6 +46,8 @@ export default class WebhookProxy {
         this.ws.on('message', (data: any) => {
             const msg = JSON.parse(data.toString());
 
+            this.logInfo(`${new Date().toISOString()} ${msg.eventType} event: ${JSON.stringify(msg)}`);
```

Comment:
Maybe modify loginfo to always put the date?

Or maybe even use the Jitsi logger which already does?